supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,320 @@
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[required-action]]
+==== image:images/yes.png[yes] Required action
+A required action is an action that a user must perform during the authentication process. A user cannot complete the authentication process until these actions are complete. For example, an admin might schedule users to reset their passwords every month. An update password required action is set for all these users.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[role]]
+==== image:images/yes.png[yes] Role
+A role identifies a type or category of user. `Admin`, `user`, `manager`, and `employee` are all typical roles that might exist in an organization. Applications often assign access and permissions to specific roles rather than individual users because dealing with users can be too granular and hard to manage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[service-account]]
+==== image:images/yes.png[yes] Service account
+Each client has a built-in service account to obtain an access token.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+//[[session]]
+==== image:images/yes.png[yes] Session
+When a user logs in, a session is created to manage the login session. A session contains information like when the user logged in and what applications have participated within single-sign on during that session. Both admins and users can view session information.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[theme]]
+==== image:images/yes.png[yes] Theme
+A theme defines HTML templates and stylesheets that you can override as you require. Every screen that Red Hat Single Sign-On provides is backed by a theme.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[user-federation-provider]]
+==== image:images/yes.png[yes] User federation provider
+Red Hat Single Sign-On can store and manage users. Often, companies already have LDAP or Active Directory services that store user and credential information. You can point Red Hat Single Sign-On to validate credentials from those external stores and pull in identity information.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[user-role-mapping]]
+==== image:images/yes.png[yes] User role mapping
+A user role mapping defines a mapping between a role and a user. A user can be associated with zero or more roles. This role mapping information can be encapsulated into tokens and assertions so that applications can decide access permissions on various resources they manage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+//[[user]]
+==== image:images/yes.png[yes] User
+A user is an entity that can log into your system. A user can have attributes associated with themselves like email, username, address, phone number, and birth day. They can be assigned group membership and have specific roles assigned to them.
```

Comment:
* "birth day" looks odd. I was going to suggest to change it to one word, but in searching it seems like "birth date" 
might be more appropriate.
* Last sentence, I'd suggest "They" -> "Users"